### ISSUE TITLE
How do I set the unit to em when I print SVG

### ISSUE THREAD

**Issue by cuixiaorui** [mathjax/MathJax#2003]
Hello, I want to know how to set up the ex transform em when output SVG. Or the ex becomes px.
I wonder if this is feasible? Sorry, I didn't find how to set it in the document.

**Comment by cuixiaorui** [comment]
![image](https://user-images.githubusercontent.com/12064746/40919590-8450c2ca-683c-11e8-8385-8363c0bb4f7e.png)

As is shown in
I want to change the unit of generated SVG from ex to em or px

**Comment by dpvc** [comment]
Can you be more specific about the use-case you have in mind?  Why do you need to use em's or px's rather than ex's?

**Comment by spiderhands** [comment]
any updates?

**Comment by bibhas2** [comment]
LibreOffice Draw does not handle "ex" unit very well. I'm having trouble importing SVG with "ex" unit. an option to use px or pt will be very good.

**Comment by dpvc** [comment]
@cuixiaorui, @spiderhands, @bibhas2, here is a version 3 configuration that uses `px` rather than `ex` for the output:

``` js
MathJax = {
  startup: {
    ready() {
      MathJax.startup.defaultReady();
      const fixed = MathJax.startup.document.outputJax.fixed;
      MathJax.startup.document.outputJax.postFilters.add(({math, document, data}) => {
        const svg = data.querySelector('svg');
        const [x, y, w, h] = svg.getAttribute('viewBox').split(/ /);
        const em = document.outputJax.pxPerEm;
        svg.setAttribute('width', fixed(w / 1000 * em) + 'px');
        svg.setAttribute('height', fixed(h / 1000 * em) + 'px');
      });
    }
  }
}
```

Perhaps that will do the trick for you.

**Comment by bibhas2** [comment]
Yeah, I ended up doing something very similar in [my app](https://bibhas2.github.io/tex-svg/). See the ``convertUnit(svg)`` method there.
